NewsFeed/views/CCBRNewsQuickViewController.m

```diff
@@ -42,6 +42,18 @@
     return [NSURL URLWithString:@"/" relativeToURL:URL].absoluteURL;
 }
 
+@interface HeaderDisplayItem : NSObject
+
+@property (nonatomic, strong) NSString *title;
+@property (nonatomic, strong) NSString *domain;
+@property (nonatomic, strong) UIImage *favicon;
+
+@end
```

Comment:
This class is not being used!

Reply:
Forgot about it, we don't need this class at the moment so i will remove it.